src/docs/community-docs/CMML/cmml-quickstart.md

```diff
@@ -46,7 +45,7 @@ The CMML creates an ARC for your project at the [DataHUB](https://git.nfdi4plant
 
 ***
 
-### 👣 Setup
+### Setup
 
 1. Please [__create a DataHUB account!__](https://register.nfdi4plants.org)
```

Comment:
I'd suggest not to mix bold font with links otherwise these are less obviously understood as links.